### ISSUE TITLE
docker images are not working

### ISSUE THREAD

**Issue by ralcaidev** [ampache/ampache-docker#54]
I've tried to deploy all three docker images, with docker-compose on two different hosts, but all I get is a 404 from apache server.

Server logs doesn't throw any relevant information.

Container inspection shows that the files are located at /var/www but they're not accesible through web browser.

My 80 port is not used.
Docker version: 19.03.13, build 4484c46
Docker-compose: 1.27.0, build 980ec85b

**Comment by m1dnight** [comment]
I have the same issue with the container. What solved it for you?

**Comment by KyleWJohnston** [comment]
I'm having this issue as well. I previously ran Ampache with a virtual host, but I have not run it as Docker. I tried Docker image ampache/ampache:4.2.5 as well as the latest but got the same result, a 404 error.

**Comment by lachlan-00** [comment]
you can see what your container is doing with logs
```
docker logs ampache
```

**Comment by KyleWJohnston** [comment]
Below are the logs after starting a container with a compose file the exact same as the default:
```
pi@raspberry:~/docker/ampache $ sudo docker logs ampache
=> Using an existing volume of MySQL
2021-01-15 01:28:00,643 INFO Set uid to user 0 succeeded
2021-01-15 01:28:00,662 INFO supervisord started with pid 1
2021-01-15 01:28:01,666 INFO spawned: 'cron' with pid 8
2021-01-15 01:28:01,672 INFO spawned: 'inotifywatch' with pid 9
2021-01-15 01:28:01,677 INFO spawned: 'mysql' with pid 10
2021-01-15 01:28:01,681 INFO spawned: 'apache2' with pid 11
2021-01-15 01:28:03,251 INFO success: cron entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2021-01-15 01:28:03,252 INFO success: inotifywatch entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2021-01-15 01:28:03,252 INFO success: mysql entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2021-01-15 01:28:03,253 INFO success: apache2 entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
```

I am running this on a Raspberry Pi (ARM), but it should be supported. Running outside of Docker has worked for me in the past on this device.

**Comment by lachlan-00** [comment]
those logs look to me like it's all running and the ports are available. but you still get a 404?

**Comment by KyleWJohnston** [comment]
Correct, I got a 404 error. I tried on a different computer running an Intel processor and Arch Linux and received the exact same logs and a 404 error.

However, I found that if I omit the last line of `docker-compose.yml` (`- ./data/sites-enabled:/etc/apache2/sites-enabled`), the webpage loads on both devices. I haven't confirmed if it is fully operational, but I'll check that next. I am running Docker as the root user - maybe that's part of the issue?
